Makefile

```diff
@@ -27,7 +28,8 @@ build:
 	$(IN_DEV) python main.py $(SPECS_FILE) \
 		-b $(BUILD_DIR) \
 		-d $(DRIVER_FILE) \
-		-r $(RUNTIME_FILE)
+		-r $(RUNTIME_LOGGING_FILE) \
+		-r $(RUNTIME_MANUAL_TASK_FILE)
```

Comment:
The two `-r` parameters is valid, when the argument is added to the parser the `action="append"` attribute is set so the two files go into an array. With the diff provided an error occurs, I believe because it thinks the second file is a positional argument.

I have a TODO to replace how this works with building the runtime and driver separately and just passing in the `.o` files which will provide more flexibility to how a runtime is compiled/what language(s) are used, etc.